otamodel/OTA_HotelDescriptiveContentNotifRQ.go

```diff
@@ -186,8 +186,8 @@ type GuaranteePaymentHotelContent struct {
 
 // PetsPolicies **
 type PetsPolicies struct {
-	PetsAllowedCode string        `json:"PetsAllowedCode"`
-	PetsPolicy      *[]PetsPolicy `json:"PetsPolicy,omitempty"`
+	PetsAllowedCode string       `json:"PetsAllowedCode"`
+	PetsPolicy      []PetsPolicy `json:"PetsPolicy,omitempty"`
```

Comment:
why you removed the pointer?
without pointer doest it remove the PetsPolicy array from output json if petpolicy missing from CM?

Reply:
Yes, it will remove PetsPolicy if it's missing from CM